test-data/unit/check-newsemanal.test

```diff
@@ -2049,3 +2049,20 @@ from pp import x as y
 def __getattr__(attr): ...
 [out2]
 tmp/a.py:2: error: Revealed type is 'Any'
+
+[case testNewAnanlyzerTrickyImportPackage]
```

Comment:
Would it be useful to an incremental mode test case as well?

Reply:
Yes, I will add an incremental test.